### ISSUE TITLE
Process not released

### ISSUE THREAD

**Issue by rgaudin** [openzim/warc2zim#132]
[youzimit task](https://farm.youzim.it/pipeline/2914a334-6886-4b97-8db7-06b87666da45) had to be canceled after 7 hours.
The crawl happened properly and warc2zim created a ZIM file but the zimit process never exited.

FYI in zimit, we call the `warc2zim.warc2zim` function and return it directly ; it's not an external process.

Please note that while in `dev`, it is not using the refactored warc2zim as `dev` is 18d old

**Comment by benoit74** [comment]
Yes, the refactored warc2zim has not yet been packaged in a new zimit image.

**Comment by benoit74** [comment]
It's still unclear how to repro the conditions of this bug.
